recipes/genomekit/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set version = "4.1.7" %}
+
+package:
+  name: genomekit
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepgenomics/GenomeKit/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9f663c750e53585411acd477a5f6954131cb402b787e85beef4753f23ac67b93
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  # tensorflow TPU whls from google only support 3.8
+  # TODO tensorflow TPU now also supports 3.10
+  skip: true  # [py<38 or py>38]
```

Let's keep it more readable
```suggestion
  skip: true  # [py!=38]
```